resources/i18n/fr.json

```diff
@@ -362,18 +362,18 @@
             "musicbrainz": "Ouvrir dans MusicBrainz"
         },
         "lastfmLink": "Lire plus...",
-        "listenBrainzLinkSuccess": "La liaison et le scrobble avec ListenBrainz sont maintenant activés pour l'utilisateur: %{user}",
-        "listenBrainzLinkFailure": "Échec lors de la liaison avec ListenBrainz: %{error}",
+        "listenBrainzLinkSuccess": "La liaison et le scrobble avec ListenBrainz sont maintenant activés pour l'utilisateur : %{user}",
```

Comment:
A couple of occurrences of the _non-breaking space before punctuation marks made of two symbols_ issue in French typography.